selfdrive/car/tests/routes.py

```diff
@@ -19,6 +19,7 @@
   FORD.ESCAPE_MK4,
   FORD.FOCUS_MK4,
   FORD.MAVERICK_MK1,
+  GM.BUICK_LACROSSE,
```

Comment:
Can you get a route on this branch, keeping it mind it will be public and for CI testing? We usually recommend starting the route/car in a parking lot, then drive around engaged for a few minutes, and ending in a parking lot as well.

Reply:
I'm still not able to have this branch recognize my car as "supported", and it goes into dashcam mode.  It does not seem to be the fault of the fingerprint, as I can move the fingerprint under the Malibu and it then thinks that my car is a Malibu and will work.  So there must be something else wrong with my modifications but I've not been able to find what it is.

Reply:
Fixed, can you unprotect your master branch so I can push to it and resolve the merge conflict?

Reply:
I have removed the branch protection rule, let me know if anything else is needed

Reply:
I took a short drive with the current code and got a radar fault. The route is 2dfdef71f033f267|2023-02-16--07-00-02

Reply:
Can you put your device on WiFi to upload logs? It sounds like there may be a communication issue somewhere, usually a bad cable.

Reply:
If you can get a route on this branch now without any faults, then I think we're ready to merge! Again a reminder that this route will be public, so a drive from parking lot, engaged, to parking lot is ideal.

Reply:
> Can you put your device on WiFi to upload logs? It sounds like there may be a communication issue somewhere, usually a bad cable.

I had the device on Wifi for a few hours this afternoon so hopefully they uploaded. If not it’ll be tonight.

Reply:
> If you can get a route on this branch now without any faults, then I think we're ready to merge! Again a reminder that this route will be public, so a drive from parking lot, engaged, to parking lot is ideal.

I’m getting way too many radar faults to be able to make a clean route.

Reply:
Make sure that all of your connections are good, even your add-a-circuit that powers the radar module.

Reply:
Did you upload rlogs for a route @koch-cf? I can check it out briefly

Reply:
I ran a test route last night with the new hardware (new C3 and new OBDII harness).  Still getting radar errors.  I've uploaded all of the files, note that this is a new dongle ID since it's new hardware:  75a6bcb9b8b40373|2023-02-25--00-12-54 .  I need some help diagnosing where these errors are coming from.  

Also note that the longitudinal tuning is also still very poor.  The car would have rear-ended the car at 19:22:30 if I had not applied the brakes at the last minute.  Both acceleration and braking are very slow, to the point where I often need to manually intervene.

Reply:
Looks like the radar messages on bus 1 are coming in inconsistently, and we're dropping messages. Total error count for the panda for can 1 is also rising

<img width="1118" alt="Screenshot_46" src="https://user-images.githubusercontent.com/25857203/221386531-97c9563d-9bd2-4a07-9e10-41975719d01b.png">

Reply:
Fixed radar error by removing the 1ft extension cable and uncoiling the USB-C cable under the dash.  Added test route 75a6bcb9b8b40373|2023-03-11--22-47-33 .